tools/pipelines/templates/build-npm-client-package.yml

```diff
@@ -40,18 +40,6 @@ parameters:
   default:
   - nyc
 
-- name: taskBundleAnalysis
-  type: boolean
-  default: false
-
-- name: taskPublishBundleSizeArtifacts
-  type: boolean
-  default: false
-
-- name: taskPack
```

Comment:
The build pipeline for the client release group always runs the packing task. This was a leftover from when we copied this template from `build-npm-client.yml` and had no use.